src/lib/mina_lib/mina_lib.ml

```diff
@@ -2056,9 +2056,6 @@ let create ?wallets (config : Config.t) =
                             valid_cb
                       | `Internal ->
                           (*Send callback to publish the new block. Don't log rebroadcast message if it is internally generated; There is a broadcast log*)
-                          don't_wait_for
-                            (Mina_networking.broadcast_state net
-                               (Mina_block.Validated.forget transition) ) ;
                           Option.iter
                             ~f:
                               (Fn.flip
```

Comment:
isn't the broadcast blocked by this validation callback? Like whatever was causing the delay to get to here, wouldn't that still affect block broadcasts?

Reply:
Nope, because there is no validation callback for internal blocks. This code does nothing. I rewrote it subtantially in Bitswap branch, but here attempted to keep the code change minimal